modules/merged_odometry_detections.py

```diff
@@ -28,17 +28,28 @@ def create(cls,
 
         return True, MergedOdometryDetections(cls.__create_key, odometry_local, detections)
 
-    def __init__(self,
-                 class_private_create_key,
-                 odometry_local: drone_odometry_local.DroneOdometryLocal,
-                 detections: "list[detections_and_time.Detection]"):
+    def __init__(
+        self,
+        class_private_create_key: object,
+        odometry_local: drone_odometry_local.DroneOdometryLocal,
+        detections: "list[detections_and_time.Detection]",
+    ) -> None:
         """
         Private constructor, use create() method.
         """
-        assert class_private_create_key is MergedOdometryDetections.__create_key, \
-            "Use create() method"
+        assert (
+            class_private_create_key is MergedOdometryDetections.__create_key
+        ), "Use create() method"
 
         self.odometry_local = odometry_local
         self.detections = detections
 
-# pylint: enable=too-few-public-methods
+    def __str__(self) -> str:
+        """
+        To string.
+        """
+        representation = (
+            f"Merged: {self.odometry_local}, detections: {len(self.detections)}\n"
+            + f"{self.detections}"
+        )
+        return representation
```

Comment:
Keep __str__() function consistent with other modules. Either just return the string or always assign representation = f"...." and return representation